### PR TITLE
[GSoC] 210-Video Recognition using SlowFast 

### PR DESCRIPTION
**Pull request by rajatkrishna**
# Video Recognition using SlowFast and OpenVINO

Added the notebook for Video Recognition using SlowFast model in this PR. Demonstrates steps to achieve the following:
- Run inference on a sample video using a ResNet based SlowFast model pretrained on the Kinetics 400 dataset and downloaded from PyTorchVideo repository
- Convert above model to OpenVINO Intermediate Representation (IR) format and serialize the same for later use
- Use OpenVINO Runtime to read, compile and run inference on the serialized model

Issue: https://github.com/openvinotoolkit/openvino_notebooks/issues/832

### CODE REVIEW

**Comment by adrianboguszewski** [comment]
Thank you for your contribution. Before we do the review please do the self-check first (tick the item when it's done):
- [ ] each function is described by docstrings and type hints
- [ ] notebook contains explicit descriptions and explanatory diagrams
- [ ] there is a README.md file in consistent style (look at other notebooks)
- [ ] the notebook is added to the main README and notebooks/README files
- [ ] there are no grammar, interpunctions or typo issues (use any free tool for that e.g. Grammarly)
- [ ] there are no committed files besides notebook and readme (please use images or videos from data dir)
- [ ] all CI checks passed

We will review only if all the above items are checked.

**Comment by rajatkrishna** [comment]
Hello @adrianboguszewski 

Thank you for the feedback. Please find the status of the self-check items below:

- [ ] each function is described by docstrings and type hints
- [ ] notebook contains explicit descriptions and explanatory diagrams
- [ ] the notebook doesn't use any data (image, video, etc.) that is not CC4.0 licensed
- [ ] there is a README.md file in consistent style (look at other notebooks)
- [ ] the notebook is added to the main README and notebooks/README files
- [ ] there are no grammar, interpunctions or typo issues (use any free tool for that e.g. Grammarly)
- [ ] there are no committed files besides notebook and readme (please use images or videos from data dir)
- [ ] all CI checks passed

**Comment by rajatkrishna** [comment]
@adrianboguszewski 

I would also like to bring to your attention that one of the CI tests has failed, which appears to be unrelated to the notebook I am currently adding. Upon further investigation, I have discovered an issue with one of the libraries utilized in that notebook to download a video from YouTube. As a result, I have submitted a fix for the problem (https://github.com/openvinotoolkit/openvino_notebooks/pull/929). However, the fix involves the addition of the mentioned video to the `notebooks/data/` folder.

I would appreciate it if you could take a moment to review the fix and provide any feedback or suggestions you may have.

**Comment by adrianboguszewski** [comment]
Thanks, @rajatkrishna for the fix :) Please rebase your work now.

We will review your PR when we have some resources. Please note we're reviewing PRs in FIFO style. 
Note: The PR must be opened before April 4th, it doesn't have to be reviewed or merged.

**Comment by rajatkrishna** [comment]
Hey @adrianboguszewski, 

Just FYI, all changes are in and rebased. CI tests are still failing due to a dependency/notebook unrelated to mine.

File: notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb [hunk 1]
```diff
@@ -0,0 +1,567 @@
+{
```
Comment (OpenVINO-dev-contest) [inline notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #38.        post_act = torch.nn.Softmax(dim=1)
</pre><p>I dont know can we use numpy instead of torch in this data process pipeline, so we can deploy openvino model without torch dependency</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (rajatkrishna) [reply notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb:1]: Replaced torch with numpy in the data process pipeline. 
The only use of torch.tensor is to feed input to the PyTorch model.

Additionally, CI tests are failing due to unrelated notebooks.

File: notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb [hunk 2]
```diff
@@ -0,0 +1,573 @@
...
```
Comment (eaidova) [inline notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    !pip install fvcore
</pre><p>please use -q for installing dependencies in quite mode</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (eaidova) [inline notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb:1]: <p>could you please add link on dataset here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by eaidova** [comment]
@rajatkrishna thank you for your contribution, it is great notebook example. Please fix my small remarks and we are ready to merge your notebook :)

**Review by eaidova — APPROVED** [review]

File: notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb (hunk 2 above)
Comment (rajatkrishna) [reply notebooks/210-slowfast-video-recognition/210-slowfast-video-recognition.ipynb:1]: Added. Thank you
